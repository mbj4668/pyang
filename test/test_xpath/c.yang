module c {
  yang-version 1.1;
  namespace "urn:c";
  prefix c;

  container x {
    leaf a {
      type int32;
    }
  }

  deviation "/x/a" {
    deviate add {
      must "../b > ../a";
    }
  }

}
