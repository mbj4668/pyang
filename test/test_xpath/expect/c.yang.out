c.yang:14: warning: XPATH_NODE_NOT_FOUND1
